definitions/3.0.0/avroSchema_v1.json

```diff
@@ -0,0 +1,317 @@
+{
```

Comment:
Out of curiosity, where did you get this schema from? https://json.schemastore.org/avro-avsc.json ? I couldn't see where it says it is 1.9 and not 1.10 or similar, that's why asking.

Reply:
I am not 100% sure, but because the AsyncAPI example is valid, I did not look further into it 😄

Reply:
So we are "maybe" giving support to Avro > 9. We recommend Avro 9 in our docs, but not sure it is a strong req.

Reply:
@dalelane as you are more into Avro world, do you think this is important?